Sources/PrimerSDK/Classes/User Interface/Apple Pay/ApplePayViewModel.swift

```diff
@@ -14,16 +14,18 @@ protocol ApplePayViewModelProtocol {
 
 class ApplePayViewModel: ApplePayViewModelProtocol {
 
-    var amount: Int {
-        guard let amount = state.settings.amount else { fatalError("Apple Pay requires amount value!") }
-        return amount
-
+    var amount: Int? {
```

Comment:
I am not sure yet whether ApplePay requires (or not) an amount and a currency. Let's refactor once we integrate it properly.